daemon/builder/dockerfile/dispatchers.go

```diff
@@ -257,11 +260,11 @@ func (d *dispatchRequest) getImageOrStage(ctx context.Context, name string, plat
 		}
 		return builder.Image(imageImage), nil
 	}
-	imageMount, err := d.builder.imageSources.Get(ctx, name, localOnly, platform)
+	imgMunt, err := d.builder.imageSources.Get(ctx, name, localOnly, platform)
```

Comment:
Was the rename here intentional?

Reply:
Oh! Yes - should've added a note; my IDE was flagging it for shadowing a type; https://github.com/moby/moby/blob/7d483021349233492c3e23609572b1e96aa576ea/daemon/builder/dockerfile/imagecontext.go#L100-L104

Reply:
LOL - the TYPO wasn't intentional though; let me fix 😂

Reply:
Fixed the typo 😄

Reply:
Ah, I assumed there was a reason just didn't see it right away. :)